workspaces/global-header/plugins/global-header/dev/index.tsx

```diff
@@ -71,7 +71,7 @@ const defaultGlobalHeaderComponentsMountPoints: GlobalHeaderComponentMountPoint[
         props: {
           icon: 'support',
           tooltip: 'Support',
-          to: '/support',
+          to: 'https://github.com/redhat-developer/rhdh/issues',
```

Comment:
When you open that page and click on create you will notice that we redirect all our users to Jira instead.

So it should be:

```suggestion
          to: 'https://issues.redhat.com/browse/RHIDP',
```

Reply:
Good catch on my dev code. Thank you 🤝 

Hmm, `https://issues.redhat.com/browse/RHIDP` is an internal Jira page, do the users have the access to it?
`https://github.com/redhat-developer/rhdh/issues` is the one anyone can visit right?

Reply:
But GH issues will be empty and users shouldn't ask questions there.

Since it's just a dev page you can maybe simple use https://developers.redhat.com/rhdh/overview ?

Reply:
This link goes directly to the create issue form if that is preferrable: https://issues.redhat.com/secure/CreateIssue.jspa?pid=12335748&issueType=13263